js/inspectShell.py

```diff
@@ -10,20 +10,9 @@
 
 import os
 import platform
-import sys
 
-path0 = os.path.dirname(os.path.abspath(__file__))
-path1 = os.path.abspath(os.path.join(path0, os.pardir, 'util'))
-sys.path.append(path1)
 import subprocesses as sps
```

Comment:
Does this work without `sys.path` foo above?

Reply:
Not sure. I wouldn't worry about it yet, there's still a bunch of sys.path stuff lying around that I'll be removing in a future PR anyway.

Reply:
Oh, wait, I misunderstood you. Yes, this was mistakenly removed, it wouldn't work without those lines.